pkg/cloudprovider/aws/fleet/capacity.go

```diff
@@ -77,9 +77,14 @@ func (c *Capacity) Create(ctx context.Context, constraints *cloudprovider.Capaci
 		}},
 	})
 	if err != nil {
-		return nil, fmt.Errorf("creating fleet, %w", err)
+		return nil, fmt.Errorf("creating fleet %w", err)
+	}
+	if len(createFleetOutput.Errors) > 0 {
+		return nil, fmt.Errorf("createFleetErrors %v", createFleetOutput.Errors)
```

Comment:
Likewise. I'd do something like "got errors while creating fleet, %v".

Now that I think a bit further will this leak instances if any were successful but we got errors?

Reply:
😲  Not sure if that can happen, will have to check, will need to either delete them or account for them some how.

Reply:
A TODO is plenty